tests/neg/i23832a.check

```diff
@@ -0,0 +1,45 @@
+-- [E120] Naming Error: tests/neg/i23832a.scala:9:8 --------------------------------------------------------------------
+9 |  given Special[Option[Int]] = ??? // error
+  |        ^
+  |        Conflicting definitions:
+  |        final lazy given val given_Special_Option: Special[Option[Long]] in object syntax at line 8 and
+  |        final lazy given val given_Special_Option: Special[Option[Int]] in object syntax at line 9
+  |---------------------------------------------------------------------------------------------------------------------
+  | Explanation (enabled by `-explain`)
+  |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  |
+  | As part of the Scala compilation pipeline every type is reduced to its erased
+  | (runtime) form. In this phase, among other transformations, generic parameters
+  | disappear and separate parameter-list boundaries are flattened.
+  |
+  | For example, both `f[T](x: T)(y: String): Unit` and `f(x: Any, z: String): Unit`
+  | erase to the same runtime signature `f(x: Object, y: String): Unit`. Note that
+  | parameter names are irrelevant.
+  |
+  | In your code the two declarations
+  |
+  |   final lazy given val given_Special_Option: Special[Option[Long]]
+  |   final lazy given val given_Special_Option: Special[Option[Int]]
```

Comment:
shouldn't it be something like:
```scala
final lazy given Special[Option[Long]] = ???
final lazy given Special[Option[Int]] = ???
```
which leads to 2 identical signatures: `given_Special_Option`?

When Developer names the `val`s, it is very obvious which names the Developer gives them - problems usually start with non-obvious auto-naming.

Reply:
The declarations are not printed with "fidelity". That could be fixed as a follow-up. (I assume but did not check that the syntax changed faster than the printer.)